tests/core/test_batch.py

```diff
@@ -0,0 +1,223 @@
+from typing import Dict
+
+import pytest
+
+from great_expectations.core.batch import (
+    BatchRequest,
+    RuntimeBatchRequest,
+    get_batch_request_from_acceptable_arguments,
+)
+from great_expectations.datasource.fluent import BatchRequest as FluentBatchRequest
+
+
+@pytest.fixture()
+def base_fluent():
+    return {"datasource_name": "ds", "data_asset_name": "da"}
+
+
+@pytest.fixture()
+def base_block(base_fluent):
+    """Basic block-style batch request args"""
+    base_fluent["data_connector_name"] = "dc"
```

Comment:
@donaldheppner ...Or we could use the constant FLUENT from the FDS module (because this is the placeholder value used for `data_connector_name` in FDS tests).  Thanks.

Reply:
I don't want to require users to provide this placeholder, so I am deliberately testing without it. Furthermore, if a data_connector_name is present, the assumes you do NOT want a Fluent Datasource.

I also have a strong preference for tiny local fixtures, I think it makes the test file much more readable.